### PR TITLE
FIX: allow Dockerfile

### LINKED ISSUE #476: binder requirements with Dockerfile?
I was trying to set up binder links for the MNE documentation with sphinx-gallery and got this error:

```py
ValueError: Did not find one of `requirements.txt` or `environment.yml` in the "dependencies" section of the binder configuration for sphinx-gallery. A path to at least one of these files must exist in your Binder dependencies.
```

Unfortunately, it's not straightforward to have headless mode work with pysurfer with just a requirements.txt / environment.yml but it's possible to do this with Dockerfile. @choldgraf do you have a sense of whether this feature can be implemented easily? I can help if so ...

### PR DESCRIPTION
**Pull request by jasmainak**
closes #476

### CODE REVIEW

**Comment by choldgraf** [comment]
Any idea why appveyor would faiil? Doesn't seem related to this... @larsoner?

**Comment by larsoner** [comment]
I've seen spurious errors in that test before, I'll make a PR and I've restarted the failed run in the meantime

File: sphinx_gallery/binder.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def check_binder_conf(binder_conf):
 
     # Ensure we have at least one dependency file
     # Need at least one of these two files
```
Comment (agramfort) [inline sphinx_gallery/binder.py]: two -> three

Comment (jasmainak) [reply sphinx_gallery/binder.py]: Fixed

**Comment by choldgraf** [comment]
Merged!

**Pull request merged**